board_size:
- 8
- 11
calibration_date: '2025-12-19 20:16:07'
camera_matrix:
  cx: 326.7016030351517
  cy: 228.33937116503992
  fx: 844.43991662607
  fy: 822.2845720465535
camera_matrix_full:
- - 844.43991662607
  - 0.0
  - 326.7016030351517
- - 0.0
  - 822.2845720465535
  - 228.33937116503992
- - 0.0
  - 0.0
  - 1.0
distortion_coefficients:
- - 0.08576713959244886
  - -5.456706441270507
  - 0.030625204091630286
  - 0.01809575919103724
  - 30.392742418844108
num_images: 35
square_size: 1.5
